docker/install-xcube-plugin.sh

```diff
@@ -29,17 +29,17 @@ elif [[ $INSTALL_MODE == "release" ]]; then
 
   cd "${PACKAGE}"-"${PACKAGE_VERSION}" || exit
 
-# xcubes python version should not be changed by the plugins
+  # xcube's python version should not be changed by the plugins
   sed -i "s/- python/#- python/g" environment.yml || exit
-# xcubes version should not be changed by the plugins
+  # xcube version should not be changed by the plugins
   sed -i "s/- xcube/#- xcube/g" environment.yml || exit
 
   cat environment.yml
-  source activate xcube && mamba env update -n xcube
-  source activate xcube && pip install .
+  micromamba env update -n base
```

Comment:
```suggestion
  micromamba install --yes --quiet --name base --file environment.yml
```

micromamba syntax is not identical to mamba, and `micromamba env update` produces an error. `install` does what we want.